packages/components/addon/components/hds/dropdown/toggle-icon.hbs

```diff
@@ -1,7 +1,7 @@
 <button class={{this.classNames}} aria-label={{this.text}} ...attributes {{on "click" this.onClick}} type="button">
   <div class="hds-dropdown-toggle-icon__wrapper">
     {{#if @imageSrc}}
-      <img src={{@imageSrc}} alt="" role="presentation" height="32" width="32" />
+      <img src={{@imageSrc}} alt="" role="presentation" />
```

Comment:
[issue] height and width should be pre-defined in the html markup for performance reasons. This tells the browser how much space to "save" for the image and makes the page faster. So even if we are declaring it in the CSS, it should still be declared here in the markup. 

```suggestion
      <img src={{@imageSrc}} alt="" role="presentation" height="32" width="32" />
```

Reply:
OK.